### PR TITLE
buildroot: Enable testing-devel

### PR DESCRIPTION
**Pull request by cgwalters**
Shorten the cycle for using new ostree in CI.

### CODE REVIEW

**Comment by cgwalters** [comment]
Motivated by https://github.com/coreos/rpm-ostree/pull/4640

**Comment by dustymabe** [comment]
any potential downsides to doing this?

File: ci/buildroot/install-buildroot.sh [hunk 1]
```diff
@@ -3,6 +3,9 @@ set -euo pipefail
 
 # This is invoked by Dockerfile
```
Comment (dustymabe) [inline ci/buildroot/install-buildroot.sh:5]: Optional: Might be useful to add a comment with rationale here for why this is useful

**Review by dustymabe — DISMISSED** [review]
Code changes LGTM

**Comment by cgwalters** [comment]
(This isn't tested by this CI, so merging)

**Pull request merged**